libs/verify-android-environment.sh

```diff
@@ -70,4 +68,7 @@ if [[ -z "${CI}" ]]; then
   fi
 fi
 
+echo "Installing uniffi_bindgen if missing; it's necessary for binding generation during the builds."
+cargo install uniffi_bindgen
```

Comment:
This is great! I ran into this dependency missing as well a few weeks ago. I think we're pinned to 0.6.1 iirc.

```suggestion
cargo install -f uniffi_bindgen --version 0.6.1
```

Reply:
Thanks, updated to use the pinned version number.